app/Console/Commands/MultiplayerEventCreate.php

```diff
@@ -0,0 +1,80 @@
+<?php
```

Comment:
All "testing" commands should be a separate pull request. And maybe as factory, not commands.